docs/tour.md

```diff
@@ -412,7 +412,7 @@ export default function(x) {
 ```
 ```js
 // app.js
-module math from "lib/mathplusplus";
+import math from "lib/mathplusplus";
```

Comment:
I believe what is actually needed in this case is this:

```js
import * as math from "lib/mathplusplus"
```

Since the `math` variable is being used to access `pi` and `e` in the later code.